### DISCUSSION TITLE
2022/automatically-send-email-when-news-is-published/

### DISCUSSION THREAD

**Discussion by giscus** [mirontoli/chuvash-eu-comments#7] [General]
# 2022/automatically-send-email-when-news-is-published/

News in SharePoint are great, it is easy to edit and post news, add pictures and web parts. But you might have thought how you would spread the word better. If your workmates would check the start page on your site or on the SharePoint Home every day, it would be awesome, it might be the case in future. In the meantime though, you probably have considered sending out email notifications to your workmates whenever there is fresh news.

https://chuvash.eu/2022/automatically-send-email-when-news-is-published/

**Comment by Danshke** [comment]
Hello there.

This solution doesn't work on mobile and outlook desktop version. The image is not showing.
Do you have a solution fot that?

Regards,
Davide

**Reply by mirontoli** [reply]
Hi! It was a while ago I had this blog post. I don't have a solution for the image problem.
